<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPlato::WorkPackageConfigPanel</class>
 <widget class="QWidget" name="KPlato::WorkPackageConfigPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>405</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Retrieve</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_CheckForWorkPackages">
        <property name="text">
         <string>Check for work packages</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Retrieve:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="kcfg_RetrieveUrl"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Archive</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="kcfg_DeleteFile">
        <property name="text">
         <string>Delete file after it has been retrieved</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kcfg_SaveFile">
        <property name="text">
         <string>Archive file after it has been retrieved</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Archive:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="kcfg_SaveUrl"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Publish</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Publish:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="kcfg_PublishUrl"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_SaveFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_SaveUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
